demo/js/demo.js

```diff
@@ -982,6 +963,16 @@ window.onload = function() {
                 // idx.SQUARE_OF_INCOME
             ],
 
+            backgroundColor: 'white',
+            gridRowProperties: [
+                undefined,
```

Comment:
Read the comments and the ticket. This is a array that represents a repeating pattern. I'll try to make it clearer in the comments.